mmv1/products/networkservices/ServiceBinding.yaml

```diff
@@ -0,0 +1,89 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'ServiceBinding'
+base_url: 'projects/{{project}}/locations/global/serviceBindings'
+create_url: 'projects/{{project}}/locations/global/serviceBindings?serviceBindingId={{name}}'
+self_link: 'projects/{{project}}/locations/global/serviceBindings/{{name}}'
+min_version: beta
+update_verb: :PATCH
+update_mask: true
+description: |
+  ServiceBinding is the resource that defines a Service Directory Service to be used in a 
+  BackendService resource.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/traffic-director/docs/reference/network-services/rest/v1beta1/projects.locations.serviceBindings'
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+    timeouts: !ruby/object:Api::Timeouts
```

Comment:
These seem very long for such a simple resource. Does this actually take a long time to provision?

Reply:
You are right, I reduced the timeouts now.